.github/workflows/publish.yml

```diff
@@ -14,6 +19,7 @@ jobs:
         uses: actions/checkout@v2
         with:
           token: ${{ secrets.BOT_GITHUB_PAT }}
+          ref: ${{ github.event.inputs.branch }}
```

Comment:
I don't think this is necessary. Actions already have a ref that points to the branch that triggered the workflow - I guess this is overriding it.

Selecting the right branch in "use workflow from" should be enough:
![image](https://user-images.githubusercontent.com/4368928/133621756-716ff049-024a-400b-ad27-fc3103a845e9.png)

The ref is also required to be set when calling the action via the github api.

Reply:
You are right!  

When I was testing I was changing the input and not the dropdown :( I'll fix this up!

Reply:
Updated vi https://github.com/grouparoo/omnibus/commit/b2eb91e17f6d96bebab11032a424d4f417774102